tests/test_app.py

```diff
@@ -35,14 +38,21 @@ def now():
 
 @responses.activate
 def test_token(client, now, enrollment, access_key):
-    responses.add(
+
+    def request_callback(request):
```

Comment:
The test would be simpler by just asserting that the `Authorization` field is sent:
```
responses.add(method='GET', url="xxx", match_querystring, json=sample_data)

assert client.get_metrics().status_code == 200
assert responses.calls[-1].request.headers['Authorization'] == "Bearer {}".format(access_key)
```